.github/workflows/cli-docs.yml

```
@@ -0,0 +1,121 @@
+          else
+            echo "No changes to commit"
+            echo "HAS_CHANGES=false" >> $GITHUB_ENV
+          fi
+
+      - name: Create PR in docs repo
+        if: env.HAS_CHANGES == 'true'
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.DOCS_REPO_TOKEN || secrets.GITHUB_TOKEN }}
+          path: docs-repo
+          commit-message: docs(cli): sync API reference ${{ steps.version.outputs.value }}
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 110: mapping values are not allowed in this context

(syntax-check)